scripts/gitlab-ci.sh

```diff
@@ -0,0 +1,451 @@
+linux-stable(){
```

Comment:
Don't we need a `#!/bin/sh` in here? (Additionally, just make sure the script in +x - not sure how to check for review)

Reply:
Not necessarily, but its good to have `#!/bin/sh`